### PR TITLE
Add Kazakh language 🇰🇿

### PR DESCRIPTION
**Pull request by drugoi**
Fixed this issue https://github.com/date-fns/date-fns/issues/1238

### CODE REVIEW

File: src/locale/kk/_lib/formatDistance/index.js [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+var formatDistanceLocale = {
+  lessThanXSeconds: buildLocalizeTokenFn({
+    regular: {
+      one: '1 секундтан аз',
+      singularNominative: '{{count}} секундтан аз',
+      singularGenitive: '{{count}} секундтан аз',
+      pluralGenitive: '{{count}} секундтан аз'
+    },
+    future: {
+      one: 'меньше, чем через секунду',
```
Comment (jarjan) [inline src/locale/kk/_lib/formatDistance/index.js]:
```suggestion
      one: 'бір секундтан кейін',
```

File: src/locale/kk/_lib/formatDistance/index.js [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+      singularNominative: 'меньше, чем через {{count}} секунду',
```
Comment (jarjan) [inline src/locale/kk/_lib/formatDistance/index.js]:
```suggestion
      singularNominative: '{{count}} секундтан кейін',
```

File: src/locale/kk/_lib/formatDistance/index.js [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+      singularGenitive: 'меньше, чем через {{count}} секунды',
```
Comment (jarjan) [inline src/locale/kk/_lib/formatDistance/index.js]:
```suggestion
      singularGenitive: '{{count}} секундтан кейін',
```

File: src/locale/kk/_lib/formatDistance/index.js [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+      pluralGenitive: 'меньше, чем через {{count}} секунд'
```
Comment (jarjan) [inline src/locale/kk/_lib/formatDistance/index.js]:
```suggestion
      pluralGenitive: '{{count}} секундтан кейін'
```

**Comment by drugoi** [comment]
Thank you for review, @jarjan!

**Comment by drugoi** [comment]
Hey, @kossnocorp, need some review.

File: src/locale/kk/index.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+import formatRelative from './_lib/formatRelative/index.js'
+import localize from './_lib/localize/index.js'
+import match from './_lib/match/index.js'
+
+/**
+ * @type {Locale}
+ * @category Locales
+ * @summary Kazakh locale.
+ * @language Kazakh
+ * @iso-639-2 rus
```
Comment (kossnocorp) [inline src/locale/kk/index.js]: Please set the tag to `kaz`

**Comment by drugoi** [comment]
Okay, @kossnocorp, I fixed language code.

File: src/locale/kk/index.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (drugoi) [reply src/locale/kk/index.js]: Already fixed, you watching for old commit.

**Comment by dkozickis** [comment]
@drugoi thank you for your work, and thank you for corrections @jarjan 🙇

**Pull request merged**